include/exit_handler/exit_handler_intel_x64_eapis.h

```diff
@@ -389,6 +397,14 @@ class exit_handler_intel_x64_eapis : public exit_handler_intel_x64
 
     exit_handler_intel_x64_eapis(const exit_handler_intel_x64_eapis &) = delete;
     exit_handler_intel_x64_eapis &operator=(const exit_handler_intel_x64_eapis &) = delete;
+
+public:
+    void handle_exit__ctl_reg_access();
+    uint64_t get_gpr_value_by_index_reg(uint64_t index_gpr);
+    void set_gpr_value_by_index_reg(uint64_t index_gpr, uint64_t new_value);
+
```

Comment:
There should be a section in the header for the handle_exit functions. Take the rest of this section and move it up so that it's under the other APIs. Near the bottom of this header we have private things and I don't want to confuse the users